### PR TITLE
add Firebase v3.0.0 with git autoupdate config

### LINKED ISSUE #7795: [Request] Add Firebase
**Library name:** Firebase 
**Git repository url:** https://github.com/firebase/firebase-bower
**npm package url(optional):** https://www.npmjs.com/package/firebase
**License(s):** https://www.firebase.com/terms/terms-of-service.html
**Official homepage:** https://www.firebase.com/
**Wanna say something? Leave message here:** I can currently pull this via jsdelivr but I would, obviously, rather use cdnjs! See their usage [here](https://www.jsdelivr.com/projects/firebase).

❤️ ❤️ 

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/33929068-request-add-firebase?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7795 
@dannyxx001 Please help me review this pull reuqest, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/firebase/firebase-bower
- Official website (optional, not the repository): https://firebase.google.com
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars: 37
  - Count of watchers: 46
  - Count of forks: 17
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [ ] The lib has notable popularity
  - [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #7795, cc @firebase

### CODE REVIEW

**Comment by dannyxx001** [comment]
@PeterDaveHello I think the PR is OK. Please help us review again, thanks.

File: ajax/libs/Firebase/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "https://github.com/firebase/firebase-bower"
+  },
+  "keywords": [
+    "Firebase",
+    "realtime",
+    "database",
+    "storage",
+    "authentication"
+  ],
+  "license": "https://developers.google.com/terms",
```
Comment (PeterDaveHello) [inline ajax/libs/Firebase/package.json]: Can use use a more precise reference here?

File: ajax/libs/Firebase/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Firebase",
+  "filename": "firebase.js",
+  "description": "Firebase Web Client",
+  "version": "3.0.0",
+  "homepage": "https://firebase.google.com",
+  "authors": "Firebase <firebase-support@google.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/firebase/firebase-bower"
```
Comment (PeterDaveHello) [inline ajax/libs/Firebase/package.json]: This seems to be a distribution purpose repo, not a real development repo.

**Comment by pvnr0082t** [comment]
I asked the question [here](https://stackoverflow.com/questions/37452399/questions-about-firebase-bower-license-and-repository). Waiting for response.

**Comment by PeterDaveHello** [comment]
Any updates here @pvnr0082t ?

**Comment by pvnr0082t** [comment]
@PeterDaveHello Is this license [reference](https://www.firebase.com/terms/terms-of-service.html) more precise? I asked for development repository, but they didn't give me an exact answer.

**Comment by PeterDaveHello** [comment]
You can use custom license and point to the url.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
@pvnr0082t How about `Unicode-TOU` as the license?
I refer to this https://libraries.io/bower/firebase
Thank you.

**Comment by sashberd** [comment]
@extend1994 @PeterDaveHello Can not rebase it to latest master. Please help to rebase in order to fix, merge and continue

**Comment by PeterDaveHello** [comment]
@sashberd this is an old PR that didn't enable to be updated by maintainers by default, let's send a new one?

**Comment by sashberd** [comment]
@PeterDaveHello No problem.

**Comment by sashberd** [comment]
@PeterDaveHello I think all PR`s older then 2017 should be closed, due to your claim above

**Pull request closed without merging**